### PR TITLE
all posts page

### PR DESCRIPTION
**Pull request by ilyaZamkovoy**

### CODE REVIEW

File: app/assets/javascripts/people.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (vladimirbazhanov) [inline app/assets/javascripts/people.coffee]: pls do not commit empty files

File: app/assets/javascripts/subscription.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (vladimirbazhanov) [inline app/assets/javascripts/subscription.coffee]: pls do not commit empty files

File: app/controllers/people_controller.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    @user_posts = @user.posts
+    subscription = current_user.subscriptions
+    @check = false
+    subscription.each do |s|
+      @check = true if s.blogger == @user.id
+    end
+  end
+
+  def index
+    @users = User.all.where("id != ?", current_user.id)
```
Comment (vladimirbazhanov) [inline app/controllers/people_controller.rb]:
```
@users = User.where.not(id: current_user.id)
```

File: app/controllers/people_controller.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  def update
+  end
+
+  def show
+    @user = User.find(params[:id])
+    @user_posts = @user.posts
+    subscription = current_user.subscriptions
+    @check = false
+    subscription.each do |s|
+      @check = true if s.blogger == @user.id
```
Comment (vladimirbazhanov) [inline app/controllers/people_controller.rb]: this is something strange

File: app/controllers/people_controller.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  def destroy
+  end
+
+  def update
+  end
+
+  def show
+    @user = User.find(params[:id])
+    @user_posts = @user.posts
+    subscription = current_user.subscriptions
```
Comment (vladimirbazhanov) [inline app/controllers/people_controller.rb]: subscription = current_user.subscriptions

wat? one object named subscription == all subscriptions?))

File: app/views/people/create.html.slim [hunk 6]
```diff
@@ -0,0 +1,2 @@
+h1 People#create
+p Find me in app/views/people/create.html.slim
```
Comment (vladimirbazhanov) [inline app/views/people/create.html.slim]: automatically generated file - pls do not add it to repo

File: app/views/people/destroy.html.slim [hunk 7]
```diff
@@ -0,0 +1,2 @@
+h1 People#destroy
+p Find me in app/views/people/destroy.html.slim
```
Comment (vladimirbazhanov) [inline app/views/people/destroy.html.slim]: and this

File: app/views/people/index.html.slim [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+        th Text
+        th Actions
+
+  - @users.each do |user|
+    - user.posts.each do |post|
+      tr
+        td= post.title
+        td= post.text
+        td.actions
+          = link_to user.full_name, user_path(user)
```
Comment (vladimirbazhanov) [inline app/views/people/index.html.slim]:
````
= link_to user.full_name, user
```

File: app/views/people/show.html.slim [hunk 9]
```diff
@@ -0,0 +1,17 @@
+.row
+  .columns
+    h2 = title(@user.full_name.to_s)
```
Comment (vladimirbazhanov) [inline app/views/people/show.html.slim]: .to_s is done automatically, you can do
```
= title user.full_name
```

File: app/views/people/show.html.slim [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+        th Title
+        th Text
+    -@user_posts.each do |post|
+      tr
+        td = post.title
+        td = post.text
+  -if @check == false
+    = link_to "follow", subscription_path(@user)
+  -else
+    = link_to "unfollow", destroysub_path(@user)
```
Comment (vladimirbazhanov) [inline app/views/people/show.html.slim]: setup you editor to add newlines to every file automatically, UNIX requires all files to have newline at the end

File: config/routes.rb [hunk 11]
```diff
@@ -1,8 +1,29 @@
 Rails.application.routes.draw do
-  resources :posts
+  get "subscription/new"
```
Comment (vladimirbazhanov) [inline config/routes.rb]: you need to use
```
resources :subscriptions
```

you should not create each route. and that routes should not be GET

File: config/routes.rb [hunk 12]
```diff
@@ -1,8 +1,29 @@
...
+
+  get "subscription/destroy"
+
+  get "subscription/show"
+
+  get "subscription/index"
+
+  get "people/new"
+
+  get "people/create"
```
Comment (vladimirbazhanov) [inline config/routes.rb]:
```
resources :people
```

File: app/controllers/people_controller.rb (hunk 4 above)
Comment (ilyaZamkovoy) [inline app/controllers/people_controller.rb]: this doing for unfollow button, here I'm checking if user follow this blogger, if not, then user see follow button

**Comment by vladimirbazhanov** [comment]
ok, lets merge, but we will discuss it when I back to office

**Pull request merged**